soco/events_asyncio.py

```diff
@@ -0,0 +1,582 @@
+# pylint: disable=not-context-manager,import-error,wrong-import-position
+
+# NOTE: The pylint not-content-manager warning is disabled pending the fix of
+# a bug in pylint. See https://github.com/PyCQA/pylint/issues/782
+
```

Comment:
This defect is closed now. Remove the disable?

Reply:
I used `events_twisted` as a base. I'll remove it and fix anything as a result